lib/types/results-wrapper.js

```diff
@@ -42,6 +45,18 @@ function getCqlObject(field) {
             return res;
         case rust.CqlType.Text:
             return field.getText();
+        case rust.CqlType.Timestamp:
+            return new Date(Number(field.getTimestamp()));
```

Comment:
Have you test for this? 
`Cql::Timestamp` hold `i64` milliseconds (max `i64` in Rust is: `9_223_372_036_854_775_807`).
But ... A JS `Date` object can represent a maximum of  `+- 8,640,000,000,000,000` milliseconds. This is less than the `Cql::Timestamp`.

Reply:
This is how does Timestamp behave in old driver:
https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/2a750ff0e11955b04161ab4a05feeda1855a827e/lib/encoder.js#L260-L262
If you believe this may be a problem, create an issue for that, and we can come back to it in second iteration